hw/ip/flash_ctrl/dv/xfile

```diff
@@ -0,0 +1 @@
+SCOPE tb.dut F
```

Comment:
Is this file needed?

Reply:
yes, https://github.com/lowRISC/opentitan/pull/9409#discussion_r759113296.